src/Database/Console/Migrations/FreshCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Hyn\Tenancy\Database\Console\Migrations;
+
+use http\Exception\RuntimeException;
+use Hyn\Tenancy\Exceptions\ConnectionException;
+use Hyn\Tenancy\Traits\MutatesMigrationCommands;
+use Illuminate\Database\Console\Migrations\FreshCommand as BaseCommand;
+
+class FreshCommand extends BaseCommand
+{
+    use MutatesMigrationCommands;
+
+    /**
+     * Execute the console command
+     */
+    public function handle()
+    {
+        try {
+            if (!$this->confirmToProceed()) {
+                return;
+            }
+
+            $this->input->setOption('force', $force = true);
+            $website_id = $this->option('website_id');
+            $realpath = $this->option('realpath');
+            $path = $this->input->getOption('path');
+
+            $website = $this->websites->query()->where('id', $website_id)->firstOrFail();
+
+            $this->connection->set($website);
+
+            $this->dropAllTables(
+                $database = $this->connection->tenantName()
+            );
+
+            $this->info('Dropped all tables related to tenant successfully');
```

Comment:
Either drop this info line or add another one that explains the full procedure was completed. Now you only provide feedback halfway.